provisioning/service/src/Query.cs

```diff
@@ -122,6 +122,64 @@ public class Query : IDisposable
             _hasNext = true;
         }
 
+#if !NET451
+        internal Query(
+            string hostName,
+            IAuthorizationHeaderProvider headerProvider,
+            string serviceName,
+            QuerySpecification querySpecification,
+            HttpTransportSettings httpTransportSettings,
+            int pageSize,
+            CancellationToken cancellationToken)
+        {
+            /* SRS_QUERY_21_001: [The constructor shall throw ArgumentNullException if the provided path is null.] */
```

Comment:
You can remove all these style comments. We've been removing them when we edit code.